packages/web/src/djitsu/components/notebook-header/index.js

```diff
@@ -73,6 +85,93 @@ export const NotebookHeader = (props) => {
     notebookRef.current = notebook
   }, [notebook])
 
+  const [showCopy, setShowCopy] = useState(false)
+
+  const CopyModal = () => {
```

Comment:
Why is this defined here?

Not sure if needs to be defined here, but if you keep it make sure to wrap it in `useMemo` otherwise it will create a new component every time the header is rendered...

Reply:
Moved it to main Render